### PR TITLE
added tests for the gen3RequestManager class

### PR DESCRIPTION
**Pull request by microbe73**

### CODE REVIEW

**Comment by microbe73** [comment]
@grugna I wrote a few tests for the basics of the Gen3RequestManager class

File: tests/test_utils.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+from Crypto.Hash import SHA256 
+import pytest
+from pcdcutils.gen3 import Gen3RequestManager
+
+# openssl genpkey -algorithm RSA -out PRIVATE_NAME.pem -pkeyopt rsa_keygen_bits:2048
+# openssl rsa -pubout -in PRIVATE_NAME.pem -out PUB_NAME.pem
+# To run the tests, create a keys directory in the tests folder and create key pairs private/public_key1 and 2.pem
+def test_successful_make_sig():
+  body = 'aaaaa'
+  pri_key_path = '/Users/ritroy/pcdcutils/tests/keys/private_key1.pem'
```
Comment (grugna) [inline tests/test_utils.py:12]: Add the keys in a tests/data folder and reference the one here for testing otherwise this will all fail

**Comment by grugna** [comment]
@microbe73 pytest is not in the pyproject.toml file, and not being in the dependency the execution of the tests can't even start. Could you update the dependencies so that they include the tests?

Also could you add a description on how to run the test in the README?

File: tests/test_utils.py (hunk 1 above)
Comment (grugna) [reply tests/test_utils.py:12]: @microbe73 you need to update the path, `/Users/ritroy/` is the user on your machine and it won't work for anyone else.

**Pull request merged**